content/influxdb/cloud/reference/release-notes/influx-cli.md

```diff
@@ -8,6 +8,20 @@ menu:
     name: influx CLI 
 ---
 
+## v2.1.1 [unreleased]
```

Comment:
@mluu12 @russorat looks like this may be v2.2.0?

Reply:
no, this will be 2.1.1.

Reply:
Thank you, Russ...saw "unreleased" v2.2.0 [here](https://github.com/influxdata/influx-cli/blob/main/CHANGELOG.md), so wanted to be sure we weren't missing anything and/or were on same page.